packages/terra-application-links/src/tabs/ApplicationTabs.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ResizeObserver from 'resize-observer-polyfill';
+import { withRouter } from 'react-router-dom';
+import 'terra-base/lib/baseStyles';
+import ApplicationTab from './_ApplicationTab';
+import ApplicationTabMenu from './_ApplicationTabMenu';
+import TabUtils from './ApplicationTabUtils';
+import styles from './ApplicationTabs.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Navigational links that will generate tabs that will update the path. These paths are matched with react-router to selection.
+   */
+  links: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.string,
+    path: PropTypes.string.isRequired,
+    text: PropTypes.string.isRequired,
+  })),
+    /**
+   * The location as provided by the `withRouter()` HOC.
+   */
+  location: PropTypes.object.isRequired,
+  /**
+   * The match as provided by the `withRouter()` HOC.
+   */
+  match: PropTypes.object.isRequired,
+  /**
+   * The history as provided by the `withRouter()` HOC.
+   */
+  history: PropTypes.object.isRequired,
+  /**
+   * The staticContext as provided by the `withRouter()` HOC. This will only be provided when
+   * within a StaticRouter.
+   */
+  staticContext: PropTypes.object,
```

Comment:
It doesn't appear the staticContext is used anywhere. Are we listing it as a prop to remove it from being passed as a customProperty?

Reply:
Yeah it's just to remove it from customProps, it comes from react-router automatically.

Reply:
Oh right. Should we look at using this approach to handle this then? https://github.com/cerner/terra-core/blob/master/packages/terra-date-picker/src/DatePicker.jsx#L242